integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/ActionConstants.java

```diff
@@ -60,25 +65,27 @@
   public static final String REMOTECONSOLE_FILE = WORK_DIR + "/console/console.jar";
   public static final String REMOTECONSOLE_DOWNLOAD_URL_DEFAULT =
       "https://github.com/oracle/weblogic-remote-console/releases/latest";
-  public static final String REMOTECONSOLE_DOWNLOAD_URL
-      =  System.getProperty("remoteconsole.download.url", REMOTECONSOLE_DOWNLOAD_URL_DEFAULT);
-  public static final String REMOTECONSOLE_VERSION  = System.getProperty("remoteconsole.version", "1.1.0");
+  public static final String REMOTECONSOLE_DOWNLOAD_URL =
+      getNonEmptySystemProperty("wko.it.remoteconsole.download.url", REMOTECONSOLE_DOWNLOAD_URL_DEFAULT);
+  public static final String REMOTECONSOLE_VERSION =
+      getNonEmptySystemProperty("remoteconsole.version", "1.1.0");
   public static final String REMOTECONSOLE_DOWNLOAD_FILENAME_DEFAULT = "console.zip";
   public static final String REMOTECONSOLE_ZIP_PATH = DOWNLOAD_DIR + "/" + REMOTECONSOLE_DOWNLOAD_FILENAME_DEFAULT;
 
+  public static final String WLE_VERSION = getNonEmptySystemProperty("wko.it.wle.version", "latest");
   public static final String WLE_DOWNLOAD_URL_DEFAULT
-      = "https://github.com/oracle/weblogic-logging-exporter/releases/latest";
+      = "https://github.com/oracle/weblogic-logging-exporter/releases/" + WLE_VERSION;
   public static final String WLE_DOWNLOAD_URL
-      = System.getProperty("wle.download.url", WLE_DOWNLOAD_URL_DEFAULT);
-  public static final String WLE_VERSION    = System.getProperty("wle.version", "latest");
-  public static final String WLE_DOWNLOAD_FILENAME_DEFAULT = "weblogic-logging-exporter.jar";
-
-  public static final String SNAKE_VERSION    = System.getProperty("snake.version", "1.27");
+      = getNonEmptySystemProperty("wko.it.wle.download.url", WLE_DOWNLOAD_URL_DEFAULT);
+  public static final String WLE_DOWNLOAD_FILENAME_DEFAULT =
+      getNonEmptySystemProperty("wko.it.wle.file.name", "weblogic-logging-exporter.jar");
+  public static final String SNAKE_VERSION =
+      getNonEmptySystemProperty("wko.it.snake.version", "1.27");
```

Comment:
I would prefer that we move version numbers to the POM.  Maybe in the second pass?